File: getTitle/php/get_title.php
<?php

namespace Morphatic;

/**
 * Gets the text from the <title> tag
 * @param  string $url The page URL
 * @return string      The <title> text
 */
function get_title($url) {
  $html = file_get_contents($url);
  preg_match('/<title>(.*?)<\/title>/', $html, $matches);
  return $matches[1];
}

File: getTitle/r/get_title.R
# get_title.R

library(httr)
library(stringr)

get_title <- function(url) {
  res <- GET(url)
  html <- content(res, "text")
  match <- str_match(
    pattern = '<title>(.*?)</title>',
    string = html
  )
  match[1,2]
}